contracts/interfaces/factory/INexusFactory.sol

```diff
@@ -24,7 +24,7 @@ interface INexusFactory {
     /// @param account The address of the newly created account.
     /// @param initData Initialization data used for the new Smart Account.
     /// @param salt Unique salt used during the creation of the Smart Account.
-    event AccountCreated(address indexed account, bytes indexed initData, bytes32 indexed salt);
```

Comment:
why did we remove indexed here?

Reply:
Removed `indexed` because it hashes `bytes` data, making it unretrievable from logs. Without `indexed`, the actual `initData` can be emitted and accessed as intended

Reply:
check L-01 for more info